Tests/fixtures/clienthints.yml

```diff
@@ -283,7 +283,7 @@
     Sec-CH-UA-Model: "KFKAWI"
     Sec-CH-Prefers-Color-Scheme: "light"
   os:
-    name: Android
+    name: Fire OS
```

Comment:
https://en.wikipedia.org/wiki/Fire_OS

need mapper base android os version for FireOS

5 android = fire os  5
7 android = fire os  6
9 android = fire os 7
10-11 android =fire os  8

Reply:
OperatingSystem::parse(): ?array

Line 316
Added code
```php

if ('HarmonyOS' === $name) {
    $version = '';
}

if ('Fire OS' === $osFromUserAgent['name']) {
        $majorVersion = (int) (\explode('.', $version, 1)[0] ?? '0');
        $fireOsVersionMapping = [
            '11'    => '8',
            '10'    => '8',
            '9'     => '7',
            '7'     => '6',
            '5'     => '5',
            '4.4.3' => '4.5.1',
            '4.4.2' => '4',
            '4.2.2' => '3',
            '4'     => '2',
            '2'     => '1'
        ];

        $version = $fireOsVersionMapping[$version] ?? $fireOsVersionMapping[$majorVersion] ?? $version;
}
```

Reply:
i update comment added 3 version full list